backend/routers/memories.py

```
@@ -113,6 +123,33 @@ def set_memory_events_state(
     return {"status": "Ok"}
 
 
+@router.patch("/v1/memories/{memory_id}/action-items", response_model=dict, tags=['memories'])
+def set_action_item_status(data: SetMemoryActionItemsStateRequest,memory_id: str,uid = Depends(auth.get_current_user_uid)):
+    memory = _get_memory_by_id(uid, memory_id)
+    memory = Memory(**memory)
```

Ruff

`Memory` may be undefined, or defined from star imports

(F405)